### PR TITLE
CreatePatientAccountActivity

### PR DESCRIPTION
**Pull request by thomaslorincz**

### CODE REVIEW

File: app/src/main/java/ca/klapstein/baudit/activities/CreateCareProviderAccountActivity.java [hunk 1]
```diff
@@ -12,50 +12,80 @@
...
+    /**
+     * Responds to account confirmation by opening the new care provider account's home screen.
+     */
     @Override
-    public void onStart() {
-        super.onStart();
+    public void onAccountConfirmed() {
+        finish();
     }
+
```
Comment (nklapste) [inline app/src/main/java/ca/klapstein/baudit/activities/CreateCareProviderAccountActivity.java]: remove unneeded newline

File: app/src/main/java/ca/klapstein/baudit/data/Email.java [hunk 2]
```diff
@@ -69,7 +69,7 @@ public int hashCode() {
         return result;
     }
 
-    @Override
+    @Override @Contract(value = "null -> false", pure = true)
```
Comment (nklapste) [inline app/src/main/java/ca/klapstein/baudit/data/Email.java]: neat!

File: app/build.gradle [hunk 3]
```diff
@@ -62,6 +62,7 @@ dependencies {
 
     // Google Maps Services
     implementation 'com.google.android.gms:play-services-maps:16.0.0'
+    implementation 'org.jetbrains:annotations-java5:15.0'
```
Comment (nklapste) [inline app/build.gradle]: could you add this libraries usage into the reuse statement

Comment (thomaslorincz) [reply app/build.gradle]: ✅

File: app/src/main/java/ca/klapstein/baudit/data/Email.java (hunk 2 above)
Comment (thomaslorincz) [reply app/src/main/java/ca/klapstein/baudit/data/Email.java]: 👨‍💻

File: app/src/main/java/ca/klapstein/baudit/activities/CreateCareProviderAccountActivity.java (hunk 1 above)
Comment (thomaslorincz) [reply app/src/main/java/ca/klapstein/baudit/activities/CreateCareProviderAccountActivity.java]: Fixed in commit b06e00e

**Pull request merged**